### DISCUSSION TITLE
Allow "fit" transform mode to function like "fill" when both dimensions are specified.

### DISCUSSION THREAD

**Discussion by RickKukiela** [craftcms/cms#12094] [Ideas] [answered]
I would like to be able to specify image transforms that fit and fill the desired height and width provided in the transform settings similar to this: https://docs.imgix.com/apis/rendering/size/fit#fill

If I request a 60x60 fit mode I would like to always get a 60x60 image but with the source image "FIT" inside of those boundaries, ie "letterboxed" (which would be transparent when possible - ie source is background transparent and the destination format supports transparency.

The existing function would persist when only one dimension is supplied.

This could be added as a new mode "fit-fill" or something to that effect in order to not break backwards compatibility.

**Comment by brandonkelly** [comment, marked as answer]
Thanks for the suggestion! This was previously requested by #8848. Follow along over there for updates.
